src/pages/subjects/Subjects.tsx

```diff
@@ -154,6 +160,7 @@ const Subjects = () => {
         {!breakpoints.isMobile && autoCompleteCategories}
         <SearchAutocomplete
           loading={subjectNamesLoading}
+          onFocus={getSubjectNames}
```

Comment:
now it fetch subjects on every focus

Reply:
yep, only when a category is changed

Reply:
> yep, only when a category is changed

changed this behavior

Reply:
You didn't fix it. As I see you still fetch the subject on every focus.